File: frontend/src/app/modules/admin/dto/bookdto.ts
import { BookStatusEnum } from "./bookstatus";

export class BookDTO{
    id?: number;
    fromDate?: Date;
    toDate?: Date;
    days?: number;
    price?: number;
    bookStatus?: BookStatusEnum;
    userId?: number;
    carId?: number;
    username?: string;
    email?: string;
}

File: backend/src/main/java/com/programmingtest/Car_Rental/dto/BookDTO.java
package com.programmingtest.Car_Rental.dto;

import com.programmingtest.Car_Rental.enums.BookStatus;
import lombok.Getter;
import lombok.Setter;


import java.util.Date;

@Getter
@Setter
public class BookDTO {

    private Long id;

    private Date fromDate;

    private Date toDate;

    private Long days;

    private Long price;

    private BookStatus bookStatus;

    private Long userId;

    private Long carId;
}